src/main/java/controller/CreateUser.java

```diff
@@ -49,7 +49,7 @@
             throw new UsernameAlreadyExistsException(path);
         } 
 
-        User newUser = new User(username);
+        User newUser = new User.Builder().username(username).build();
```

Comment:
Please check if we have User.newBuilder(). instead?

Reply:
here and elsewhere

Reply:
I have made a static class named Builder inside the User class, which I am using for creating User instances. I did not understand what you mean by `newBuilder()`.

Reply:
You should make the constructor of `Builder` private and instead expose a public static method in `User` called `newBuilder` which calls the constructor and returns a new instance.

Reply:
Got it, Done.